docs/source/user-guide/user_interface/integrations.rst

```diff
@@ -0,0 +1,122 @@
+=============
+Integrations
+=============
+
+
+Notifications
+=============
+
+
+Email Notifications
+-------------------
+
+To configure email notifications go to the page 'Configurations/Integrations' and click on the tab 'Notifications'.
+On the shown table mark alert severity levels about which you want to be notified.
+
+
+.. image:: /_static/images/user-guide/user_interface/8.1.1_selection_of_alert_severity_for_email_notifications.png
+    :width: 600
+
+
+Example of an email notification.
+
+
+.. image:: /_static/images/user-guide/user_interface/8.1.1_example_of_email_notification.png
+    :width: 600
+
+
+
+Slack Notifications
+-------------------
+
+To configure Slack notifications go to the page 'Configurations/Integrations' and click on the tab 'Notifications'.
```

Comment:
@yromanyshyn change to 'Notifications' tab

Reply:
fixed